.gitignore

```diff
@@ -3,3 +3,5 @@
 target/
 .DS_Store
 .idea/
+.floo
+.flooignore
```

Comment:
Just out of curiosity, what are floo files for?

Reply:
Its for an addon I have for Sublime text called Floobits